fastlane/Fastfile

```diff
@@ -102,6 +102,27 @@ end
 
 desc "Installs all Certs and Profiles necessary for development and ad-hoc"
 lane :match_me do
+  run_match_me(readonly: true)
+end
+
+desc "Register new device, regenerates profiles, updates them remotely and locally"
+lane :register_new_device do
+  device_name = prompt(text: "Enter the device name: ")
+  device_udid = prompt(text: "Enter the device UDID: ")
```

Comment:
If you wanna be fancy, you can print `xcrun xctrace list devices ` where this lists connected devices (and simulators :( ) with their UDIDs for faster copy-past action

Reply:
I have an infinite list of simulators and getting the device UDID is simpler by Finder, IMO 🤔
@b-onc WDYT?

Reply:
Okay, you can do this:

`xcrun instruments -s devices | grep -v Simulator`, this will show only physical devices, however xcrun instruments is deprecated now...